src/containers/Tag/TagList/TagList.tsx

```diff
@@ -21,11 +21,16 @@ import { GET_TAGS } from '../../../graphql/queries/Tag';
 import { NOTIFICATION } from '../../../graphql/queries/Notification';
 import { DELETE_TAG } from '../../../graphql/mutations/Tag';
 import { ToastMessage } from '../../../components/UI/ToastMessage/ToastMessage';
+import DialogBox from '../../../components/UI/DialogBox/DialogBox';
```

Comment:
This will be` import {DialogBox} from '../../../components/UI/DialogBox/DialogBox';`. so that we follow same convention.